src/main/java/com/atlassian/bitbucket/jenkins/internal/credentials/JenkinsToBitbucketCredentialsImpl.java

```diff
@@ -17,7 +18,12 @@
 
     @Override
     public BitbucketCredentials toBitbucketCredentials(@Nullable String credentialId) {
-        return CredentialUtils.getCredentials(credentialId)
+        return toBitbucketCredentials(credentialId, null);
+    }
+
+    @Override
+    public BitbucketCredentials toBitbucketCredentials(@Nullable String credentialId, @Nullable Item context) {
```

Comment:
Ditto in this method. Apart from a lot of test usages, there are two methods that still use jenkins-scoping for Bitbucket credentials:
1. Webhook handler. In the typical case, this should be using the global admin credential which is always jenkins-scoped so that's fine, but I think we can still explicitly state the context
2. The mirror resolver, and I'm actually wondering if this is also a bug- it uses job credentials. I haven't tested it, but I expect this to fail if the job is in a folder. We can address this by adding the conext to the `MirrorFetchRequest`, but I don't think we need to do that in this PR.

Maybe leave this for now, but I think ultimately it would be good to remove the nullability annotations from these methods too.

Reply:
Yeah it's a bug :( Raised in JENKINS-66502, outside the scope of this PR.